spec/integration/v5_full_assessment_spec.rb

```diff
@@ -244,11 +258,40 @@ def vehicle_params
           "loan_amount_outstanding" => "0.0",
           "date_of_purchase" => "2020-08-18",
           "in_regular_use" => true,
+          "subject_matter_of_dispute" => true,
```

Comment:
I know it's probably outside of the scope of this change, but it worries me that this vehicle was purchased 2 months after the submission date in the test. Might it be sensible to do a separate 'test cleanup' PR to sort out this (and change the submission date to 2022)? WDYT?

Reply:
Ah, but in this case the submission date in the test _is_ now 2022, so the date of purchase is in the past. But yes, I think a separate PR to tidy up the tests and deal with the age of the vehicle issue is probably a good thing.